app/src/components/AddProposalPanel.js

```diff
@@ -1,56 +1,81 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import { Button, Field, TextInput } from '@aragon/ui'
 import styled from 'styled-components'
 
 const AddProposalPanel = ({ onSubmit }) => {
-  const [title, setTitle] = useState('')
-  const [link, setLink] = useState('')
-  const [amount, setAmount] = useState(0)
-  const [beneficiary, setBeneficiary] = useState('')
-  const disabled = false // TODO Disable when empty or invalid fields
+  const [form, setForm] = useState({
+    title: '',
+    link: '',
+    amount: 0,
+    beneficiary: '',
+    description: '',
+  })
+  const [isDisabled, setStatus] = useState(true)
+
+  const isFormValid = form => form.filter(i => i === '' || i === 0).length === 0
+
+  useEffect(() => {
+    const values = Object.values(form)
+    if (isFormValid(values)) return setStatus(false)
+    return setStatus(true)
+  }, [form])
+
   const onFormSubmit = event => {
     event.preventDefault()
-    onSubmit({ title, link, amount, beneficiary })
+    onSubmit(form)
   }
+
   return (
     <Form onSubmit={onFormSubmit}>
       <Field label="Title">
         <TextInput
-          onChange={event => setTitle(event.target.value)}
-          value={title}
+          onChange={event => setForm({ ...form, title: event.target.value })}
+          value={form.title}
           wide
           required
         />
       </Field>
+      <Field label="Description">
+        <TextInput
+          onChange={event =>
+            setForm({ ...form, description: event.target.value })
+          }
+          value={form.description}
+          wide
+          multiline
+        />
+      </Field>
```

Comment:
We can't store proposal descriptions yet (as they should be stored and pinned on IPFS, and we are not ready for that). I would remove this field for now.